src/test/java/wooteco/subway/domain/SectionsTest.java

```diff
@@ -0,0 +1,188 @@
+package wooteco.subway.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static wooteco.subway.Fixtures.GANGNAM;
+import static wooteco.subway.Fixtures.HYEHWA;
+import static wooteco.subway.Fixtures.JAMSIL;
+import static wooteco.subway.Fixtures.SECTION_1_2;
+import static wooteco.subway.Fixtures.SECTION_1_3;
+import static wooteco.subway.Fixtures.SECTION_2_3;
+import static wooteco.subway.Fixtures.SECTION_3_4;
+import static wooteco.subway.Fixtures.SINSA;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@Nested
+public class SectionsTest {
+
+    @Test
+    @DisplayName("구간들에서 역들을 추출한다.")
+    void toStationIds() {
+        final Sections sections = new Sections(new Section(new Station(1L, HYEHWA), new Station(2L, SINSA), 10),
+                new Section(new Station(2L, SINSA), new Station(3L, GANGNAM), 10));
+
+        final List<Station> stations = sections.toSortedStations();
+
+        assertThat(stations).containsExactly(new Station(1L, HYEHWA), new Station(2L, SINSA), new Station(3L, GANGNAM));
+    }
+
+    @Nested
+    @DisplayName("지하철 구간을 추가한다.")
+    class AddSectionTest {
+
+        @Test
+        @DisplayName("1-2 구간이 있을 때, 2-3 구간을 추가한다.")
+        void addSection1() {
+            // given
+            final Sections sections = new Sections(List.of(SECTION_1_2));
```

Comment:
```suggestion
            final Sections sections = new Sections(SECTION_1_2);
```
가변 인자를 사용하는 생성자를 만들어두셨는데요, 활용해도 좋을 것 같습니다 😄

Reply:
으아앗 일부러 가변 인자를 받는 생성자를 추가해 놓고 잘 활용하지 못하고 있었네요😅 전체적으로 변경했습니다!!